wifiphisher/common/interfaces.py

```diff
@@ -1,163 +1,147 @@
-#pylint: skip-file
 """
-This module was made to handle all the interface related operations for
-Wifiphisher.py
+This module was made to handle all the interface related operations of
+the program
 """
 
 import pyric
 import pyric.pyw as pyw
 
 
-class NotEnoughInterfacesFoundError(Exception):
-    """
-    Exception class to raise in case of a finding less than enough interfaces.
-    """
```

Comment:
Of'course the check would be if there are two wireless interfaces (`pyric.iswireless`) installed.